test/hexpm/accounts/web_auth_test.exs

```diff
@@ -0,0 +1,101 @@
+defmodule Hexpm.Accounts.WebAuthTest do
+  use Hexpm.DataCase, async: true
+
+  alias Hexpm.Accounts.WebAuth
+
+  @scope "write"
+
+  describe "get_code/1" do
+    test "returns a valid response on valid scope" do
+      for scope <- ["write", "read"] do
+        response = WebAuth.get_code(scope)
+
+        assert response.device_code
+        assert response.user_code
+      end
+    end
+
+    test "returns an error on invalid scope" do
+      {status, _changeset} = WebAuth.get_code("foo")
+      assert status == :error
+    end
+  end
+
+  describe "submit/3" do
+    setup [:get_code, :login]
+
+    test "returns ok on valid params", c do
+      audit_data = audit_data(c.user)
+
+      {status, _changeset} = WebAuth.submit(c.user, c.request.user_code, audit_data)
```

Comment:
```suggestion
      assert {:ok, ...} = WebAuth.submit(c.user, c.request.user_code, audit_data)
```

please assert what `...` is. If it is unused in production code too, let's just return `:ok` from this function.

Reply:
`...` is the response from `Repo.update` in the case of valid params.

We might have a `{:error, "invalid user code"}` in the case of invalid params.